modules/iqzoneBidAdapter.js

```diff
@@ -0,0 +1,176 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+
+const BIDDER_CODE = 'iqzone';
+const AD_URL = 'https://smartssp-us-east.iqzone.com/pbjs';
+
+function isBidResponseValid(bid) {
+  if (!bid.requestId || !bid.cpm || !bid.creativeId ||
+        !bid.ttl || !bid.currency) {
+    return false;
+  }
+
+  switch (bid.mediaType) {
+    case BANNER:
+      return Boolean(bid.width && bid.height && bid.ad);
+    case VIDEO:
+      return Boolean(bid.vastUrl || bid.vastXml);
+    case NATIVE:
+      return Boolean(bid.native && bid.native.impressionTrackers && bid.native.impressionTrackers.length);
```

Comment:
I am trying to debug Native example and in the response, apart from `bid.cpm` missing, there's also this line `bid.native.impressionTrackers` that is empty.

Reply:
We fixed all problems